wiki/Tournaments/The_Roundtable/16_player_battle_royale/en.md

```diff
@@ -0,0 +1,64 @@
+---
+tags:
+  - The Round table
+---
+
+# The Roundtable 16 player battle royale
+
+![The Roundtable banner](/wiki/shared/news/2022-07-08-the-roundtable/roundtable_banner.png)
+
+[**The Roundtable**](/wiki/Tournaments/The_Roundtable)'s **16 player battle royale** takes place the first day of the event.
```

Comment:
this intro paragraph of each tournament should explain what it actually is / how it's played

if you wanna get this PR in quickly cuz news post tho, this can be addressed in a later update